src/js/index.js

```diff
@@ -43,7 +47,7 @@ class Toast {
       this.toast.className += ' native-toast-edge'
     }
 
-    if (square) {
+    if (!square) {
       this.toast.style.borderRadius = '3px'
```

Comment:
I've tried on edge mode so you're right. on edge mode `borderRadius` always should not be set. I've add a new commit to fix that.